back/src/config/pg/kysely/model/database.ts

```diff
@@ -378,12 +378,12 @@ interface OngoingOauths {
 
 interface Establishments extends WithAcquisition {
   additional_information: Generated<string | null>;
+  is_monthly_discussion_limit_reached: Generated<boolean>;
```

Comment:
comme discuté sur discord, à mon avis c'est mieux d'éviter la notion de `monthly` car ce critère change régulièrement.

```suggestion
  max_discussions_for_period_reached: Generated<boolean>;
```